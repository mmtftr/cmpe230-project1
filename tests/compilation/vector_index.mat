vector _dsajdlkas[32]
_dsajdlkas[14] = 2

print(_dsajdlkas[14])

_dsajdlkas[32] = 1342.32
print(_dsajdlkas[32])

print(_dsajdlkas[1])
