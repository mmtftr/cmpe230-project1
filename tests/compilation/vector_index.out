2
1342.320000
0